controllers/domachine_controller.go

```diff
@@ -194,6 +195,29 @@ func (r *DOMachineReconciler) Reconcile(req ctrl.Request) (_ ctrl.Result, reterr
 	return r.reconcile(ctx, machineScope, clusterScope)
 }
 
+func (r *DOMachineReconciler) reconcileVolumes(ctx context.Context, mscope *scope.MachineScope, cscope *scope.ClusterScope) (reconcile.Result, error) {
+	mscope.Info("Reconciling DOMachine Volumes")
+	computesvc := computes.NewService(ctx, cscope)
+	for _, disk := range mscope.DOMachine.Spec.DataDisks {
```

Comment:
If you agree with updating the for loop to range DataDisks using `domachine` in `reconcileDeleteVolumes` (other comment).

Then maybe we can do that here too so that we can use it in 2 places:
```
domachine := mscope.DOMachine
for _, disk := range domachine.Spec.DataDisks {
    volName := infrav1.DataDiskName(domachine, disk.NameSuffix)
```

Reply:
done